test/app/test_run.rb

```diff
@@ -168,14 +168,8 @@ module Roby
                 out, = capture_subprocess_io do
                     dir = roby_app_setup_single_script
                     pid = roby_app_spawn("run", "does_not_exist.rb", chdir: dir)
-                    roby_app_with_polling(timeout: 10, period: 0.1) do
-                        _, status = Process.waitpid2(pid, Process::WNOHANG)
-                        if status
-                            assert status.exited?
-                            assert_equal 1, status.exitstatus
-                            true
-                        end
-                    end
+                    status = assert_roby_app_exits(pid)
```

Comment:
`assert_roby_app_exists` timeout is 5s and here 10s was used. This is ok because in practice we never achieve the timeout?

Reply:
Another commit changed all timeouts to 10s